examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.ts

```diff
@@ -48,7 +48,7 @@ export class SubjectController {
   async updateBpiSubject(
     @Param('id') id: string,
     @Body() requestDto: UpdateBpiSubjectDto,
-  ): Promise<void> {
+  ): Promise<UpdateBpiSubjectDto> {
```

Comment:
this is request dto, i dont think we should use it as response dto too
i think we should keep PR related to mapping changes only to avoid non related changes